base_domain_operator/README.rst

```diff
@@ -0,0 +1,77 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+=======================
+Custom domain operators
+=======================
+
+This module was written to allow developers to define custom domain operators. This can be very helpful for complicated selections or ir.rules.
+
+Currently implemented operators
+===============================
+
+``parent_of``
+  The inverse of ``child_of``: Searches for the parents of given records in the hierarchy, including the records themselves.
```

Comment:
Include here a comment saying that the operator is similar to the v10 one and can be removed when migrating the module to that version.